gapis/trace/android/trace.go

```diff
@@ -171,12 +171,13 @@ func (t *androidTracer) Validate(ctx context.Context) error {
 		doneSignal, doneFunc := task.NewSignal()
 
 		crash.Go(func() {
+			// TODO(b/142824856): This is a workaround to a problem that render stages data
```

Comment:
If this is not fixed soon, we might want to document it to the users?

Reply:
The users don't need to know this I think.